salt/_auth/kubernetes_rbac.py

```
@@ -184,22 +236,10 @@ def auth(username, password=None, token=None, **kwargs):
     return result
 
 
+@_check_auth_args(on_fail=lambda: [])
```

When calling with an incorrect combination of arguments, shouldn't this `raise` instead of return `[]` as it does now?

---

Makes sense, and would make the code simpler :) I blindly followed the previous approach

---

Done